### PR TITLE
adds flag for custom host

### LINKED ISSUE #250: Ask if we should run tunnel
# Issue summary

We currently start a tunnel on app create and assume the user wants it. Often they have tunnels running on their own it causes it to error out, or they need to update an address in the admin and they may not want to. We should ask before we run the ngrok tunnel.

### PR DESCRIPTION
**Pull request by katiedavis**
### WHY are these changes introduced?

Fixes #250  

### WHAT is this pull request doing?

Allows the user to pass a custom `--host` flag. Does not run tunnel if this is the case.

### CODE REVIEW

File: lib/shopify-cli/commands/serve.rb [hunk 1]
```diff
@@ -6,12 +6,22 @@ module ShopifyCli
...
 
       def call(*)
         project = Project.current
-        url = ShopifyCli::Tasks::Tunnel.call(@ctx)
+        custom_host = options.flags[:host]
+        url = if custom_host
+            custom_host
+        else
+          ShopifyCli::Tasks::Tunnel.call(@ctx)
+        end
```
Comment (tylerball) [inline lib/shopify-cli/commands/serve.rb]: this can be simplified to `url = options.flags[:host] || Tasks::Tunnel.call(@ctx)`

**Comment by katiedavis** [comment]
> We should add another test for running serve with this arg

Yes totally! this is still WIP I've been having issues with tests

File: lib/shopify-cli/commands/serve.rb (hunk 1 above)
Comment (katiedavis) [reply lib/shopify-cli/commands/serve.rb]: Oh yes thanks!

**Comment by barda999** [comment]
Is there any documentation about how to use this --host flag? I cannot find it anywhere, also looked at [https://shopify.dev/apps/tools/cli/commands#dev-flags](https://shopify.dev/apps/tools/cli/commands#dev-flags)

**Pull request merged**